.github/workflows/danger.yml

```diff
@@ -0,0 +1,20 @@
+name: PR Linter
+on: [pull_request]
+jobs:
+  danger:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_GEMFILE: ${{ github.workspace }}/Gemfile.danger
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
It is my understanding that this will fetch all the commits as opposed to simply the last one. I'm not familiar with Danger though so I'm unsure why this is valuable. Can you illuminate?

Reply:
I forget why that was needed, but it's fetch all history for all tags and branches. I think there was some error from Danger when it didn't have access to all of it.